app/tests/util/connections.py

```diff
@@ -260,3 +299,44 @@ async def fetch_or_create_trust_registry_connection(
             verifier=verifier,
             connection_alias=connection_alias,
         )
+
+
+async def create_did_exchange(
+    bob_member_client: RichAsyncClient, alice_member_client: RichAsyncClient, alias: str
+) -> BobAliceConnect:
+
+    # Get Bob's public DID
+    bob_public_did = (await bob_member_client.get(f"{DID_BASE_PATH}/public")).json()[
+        "did"
+    ]
```

Comment:
Bad style - it's an await, .json, and reading key all in one line. Do await and then .json()[key] separately

It's mainly that doing `(await ..).method()` looks bad, but if it's _just_ (await ..).json() then it's fine, since it happens so often. Chaining more operations in one line after that it starts being hard to read

Reply:
P.S. Add a note that bob is assumed to have a public did, so "bob" is an issuer in this context